### PR TITLE
F22 amendments

### PR DESCRIPTION
**Pull request by intermezzio**
Combining all changes from the Fall 2022 town hall into one, more concise pull request.

### CODE REVIEW

**Review by ndhulipala1 — APPROVED** [review]
Approved through general vote at the Fall 2022 Town Hall.

**Review by ndhulipala1 — APPROVED** [review]
Approved through general vote at Fall 2022 Town Hall.

File: honorcode.md [hunk 1]
```diff
@@ -38,10 +38,10 @@ College.
 I will be receptive to change, supportive of innovation, and willing to take
 risks for the benefit of the community.
 ### 3.5 Do Something
-I will strive to better myself and my community and take responsibility for my
-own behavior. When I become aware of a violation of the Honor Code or an issue
-within the community, I will take action towards resolution of the situation. I
-expect others to do the same.
+I will strive to be an active advocate for the wellbeing of my community. I will
```
Comment (syclops) [inline honorcode.md:41]: wellbeing -> well-being

File: honorcode.md [hunk 2]
```diff
@@ -38,10 +38,10 @@ College.
...
+seek to understand and then act on issues I perceive around me that are specific
```
Comment (syclops) [inline honorcode.md:42]: understand and then act on -> understand, and then act on,

File: honorcode.md [hunk 3]
```diff
@@ -38,10 +38,10 @@ College.
...
+to both Olin and beyond. I will engage in open, self-reflective discussion with
```
Comment (syclops) [inline honorcode.md:43]: to both Olin and beyond -> to Olin, as well as beyond

**Review by syclops — APPROVED** [review]
Some minor fixes, but looks good to me.

File: honorcode.md (hunk 3 above)
Comment (intermezzio) [reply honorcode.md:43]: I can't change the actual wording because that's what was voted on, so I only fixed typos

Comment (intermezzio) [reply honorcode.md:43]: This can be addressed in a future town hall though

**Review by syclops — APPROVED** [review]

**Pull request merged**